s3_sync/s3_sync.py

```diff
@@ -0,0 +1,158 @@
+	os.environ['slack_webhook_url'] = config['Slack']['webhook_url']
+
+def verify_local_folder_exists(local_video_records_done_folder):
+	"""
+    Check whether the local video records done folder exists
+
+    Args:
+      local_video_records_done_folder (str): The path of the local video
+      	records done folder.
+
+    Returns:
+      If folder exists return None, and if not, logs error and exit.
+	"""
+	if os.path.exists(local_video_records_done_folder):
+		return
+	else:
+		logger.exception("Local Video Records Done folder not found")
+		sys.exit("[-] Local Video Records Done folder not found")
+
+def verify_aws_cli_installed(aws_cli_binary):
+	"""
+	Check whether AWS CLI is installed
+
+	Args:
+	  aws cli binary (str): absolute path to aws cli binary file.
+
+	Returns:
+	  If file exists, return None, else log error and exit.
+
+	"""
+	if os.path.exists(aws_cli_binary):
+		return
+	else:
+		logger.exception("Could not find AWS CLI executable:")
+		sys.exit("[-] Could not find AWS CLI executable")
+
+def verify_s3_bucket_exists(s3_bucket_name):
+	"""
+	Check whether S3 bucket exists
+
+	Args:
+	  s3_bucket_name (str): The s3 bucket name
+
+	Returns:
+	  list: if connection established and bucket found, return list of
+	  	objects in bucket otherwise error and exit on any issues trying
+	  	to list objects in bucket.
+	"""
+	try:
+		ls_s3_bucket_cmd = 'aws s3 ls {}'.format(s3_bucket_name)
+		subprocess.run(ls_s3_bucket_cmd, shell=True, check=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+		return
+	except subprocess.SubprocessError:
+		logger.exception("Failed to list specified s3 bucket")
+		sys.exit("[-] Failed to list specified s3 bucket")
+
+def notify_slack_channel(slack_message):
+	"""
+	Send notification to Slack Channel
+
+	Args:
+	  slack_message (str): message to send to slack
+	"""
+	try:
+		resp = requests.post(os.environ.get('slack_webhook_url'), json={"text": slack_message,
+			"username": config['Slack']['bot_username'],
+			"icon_emoji": config['Slack']['bot_emoji'],})
+		return
+	except (requests.exceptions.RequestException, NameError) as err:
+		logger.warn("Failed to notify slack channel with following error: {}", err)
+		return
+
+def sync_local_to_s3(local_video_records_done_folder, s3_bucket_name):
+	"""
+	Sync local files to specified S3 bucket
+
+	Args:
+	  local_video_records_done_folder (str): local folder containing video files
+	   	ready to be copied to S3.
+	  s3_bucket_name (str): s3 bucket name
+	"""
+	try:
+		s3_sync_cmd = 'aws s3 sync {local_video_records_done_folder} "s3://"{s3_bucket_name}'.format(
+			local_video_records_done_folder=local_video_records_done_folder, s3_bucket_name=s3_bucket_name)
+		cmd_output = subprocess.run(s3_sync_cmd, shell=True, check=True, stdout=subprocess.PIPE,
+			stderr=subprocess.PIPE)
+		logger.info("S3 sync successfully ran: {}", cmd_output)
+		notify_slack_channel("Sync succeeded on: " + "*" + computer_name + "*" + "\n" + "`" + str(cmd_output) + "`")
```

Comment:
If we can enforce a Python 3.6 runtime then you can use fstrings (https://docs.python.org/3/reference/lexical_analysis.html#f-strings)